api/api/spec/spec.yaml

```diff
@@ -2555,6 +2560,7 @@ components:
       schema:
         type: integer
         format: int32
+        minimum: 1
```

Comment:
I have added minimum 1 for the ID so to avoid errors with 0 and negative numbers (non existent)